MediaBrowser.Controller/Syncplay/ISyncplayManager.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using MediaBrowser.Controller.Session;
+using MediaBrowser.Model.Syncplay;
+
+namespace MediaBrowser.Controller.Syncplay
+{
+    /// <summary>
+    /// Interface ISyncplayManager.
+    /// </summary>
+    public interface ISyncplayManager
+    {
+        /// <summary>
+        /// Creates a new group.
+        /// </summary>
+        /// <param name="session">The session that's creating the group.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        void NewGroup(SessionInfo session, CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Adds the session to a group.
+        /// </summary>
+        /// <param name="session">The session.</param>
+        /// <param name="groupId">The group id.</param>
+        /// <param name="request">The request.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        void JoinGroup(SessionInfo session, string groupId, JoinGroupRequest request, CancellationToken cancellationToken);
```

Comment:
Why is `groupId` a string instead of a `Guid`? (Not just here, but almost everywhere)

Reply:
Good point. I'll update these.